Dump/run_otg_proc.py

```diff
@@ -66,12 +68,12 @@ def run_otg_proc():
 #--- set temp file 
 #
     tmp_file = work_dir + 'gratstat.in.tl'
```

Comment:
Consider using consistently the following way of joining strings: tmp_file = f"{work_dir}/gratstat.in.tl"

Reply:
I plan on addressing the string consistency in total once I revisit this set of script for the PR which will include the tests I wrote. Should clear up this kind of issue with the entire set at once.

Reply:
OK